src/routes/data-decode/__tests__/get-data-decoded.dto.factory.ts

```diff
@@ -0,0 +1,9 @@
+import { faker } from '@faker-js/faker';
+import { GetDataDecodedDto } from '../entities/get-data-decoded.dto.entity';
+
+export default function (data?: string, to?: string): GetDataDecodedDto {
```

Comment:
Renamed `CreateDataDecodedDto` to `GetDataDecodedDto`. I think it makes more sense from the clients perspective.

Reply:
Why don't we use the `Builder` class?

Reply:
I deleted this file in [3a1de96](https://github.com/5afe/safe-client-gateway-nest/pull/278/commits/3a1de96b74e40438d80e3a094f6e635d7330182d) since it was used by a redundant test class only (see https://github.com/5afe/safe-client-gateway-nest/pull/278#discussion_r1115971367)